lib/stages/splitMergedBeamBuffer.hpp

```diff
@@ -0,0 +1,65 @@
+
+/**
+ * @file
+ * @brief Split the merged beam buffer to single framed buffer.
+ *  - splitMergedBeamBuffer : public kotekan::Stage
+ */
+
+#ifndef SPLITE_MERGED_BEAM_BUFFER
+#define SPLITE_MERGED_BEAM_BUFFER
+
+#include "BeamMetadata.hpp" // for BeamMetadata
+#include "Config.hpp"       // for Config
+#include "Stage.hpp"
+#include "bufferContainer.hpp" // for bufferContainer
+
+#include <stdint.h> // for uint32_t
+#include <string>   // for string
+#include <vector>   // for vector
+
+
+using std::vector;
+
+
+/**
+ * @class splitMergedBeamBuffer
+ * @brief This stage splits the merged beam buffer to a single frame beam buffer.
+ *
+ * @par Buffers
+ * @buffer in_buf Kotekan buffer of merged packets.
+ *     @buffer_format Array of @c chars
+ * @buffer out_buf Kotekan buffer of single frame beam packets.
+ *     @buffer_format Array of @c uint32_t
+ *
+ * @author Jing Santiago Luo
+ *
+ *
+ */
+
+
+class splitMergedBeamBuffer : public kotekan::Stage {
```

Comment:
There are a lot of extra blank lines here and in other places.   Generally it's best to only use blank lines where it adds in readability, and normally no more than one at a time. 
```suggestion
 * @author Jing Santiago Luo
 */
class splitMergedBeamBuffer : public kotekan::Stage {
```